### DISCUSSION TITLE
アーティファクト装備調整案について (3.0.0α→3.0.0β)

### DISCUSSION THREAD

**Discussion by iks3** [hengband/hengband#279] [Ideas]
3.0.0αまでに実装されたアーティファクト装備の調整についてのスレッドです。

**Comment by iks3** [comment]
[255] ★ドン・キホーテのヘヴィ・ランス (3d8) (-12,+12) / The Heavy Lance of Don Quixote について
I:22:29:-3
W:40:30:300:70000
P:0:3d8:-12:12:0
F:STR | CON | CHR | IM_ELEC | RES_DARK | RES_FEAR | RIDING
F:FREE_ACT | SUST_STR | SUST_CON | SUST_CHR | SLAY_EVIL | KILL_GIANT

**Reply by iks3** [reply]
以下のような感じでいかがかと思います。隠密もマイナスでもいいかなとは思わなくもないですが。
(3d8) (-13,+13) (-2) {+器魅*電r恐;遅/Xト巨/邪~巨ト} 発動:狂戦士化

I:22:29:-2
W:40:30:300:76543
P:0:3d8:-13:13:0
F:DEX | CHR | RIDING | SLOW_DIGEST | ACTIVATE
F:IM_ELEC | RES_FEAR | ESP_GIANT | ESP_TROLL
F:SLAY_EVIL | KILL_GIANT | KILL_TROLL
U:BERSERK

**Reply by iks3** [reply]
-1 知賢器魅隠探の方が良さそうなのでその方向で。

**Reply by Aquashrimp** [reply]
俺もこのぐらいのペナルティで良いかなと思います

**Reply by iks3** [reply]
α9で実装しました。

**Comment by iks3** [comment]
### 武器編
|名称|変更点|対応|
|---|---|---|
|ナルサンク|(2d4)、+1腕力、発動FIRE_BALL_AND_RESITANCE|α9|
|ニムサンク|(2d4)、+1器用、発動COLD_BALL_AND_RESITANCE|α9|
|デサンク|(2d4)、+1耐久、発動ELEC_BALL_AND_RESISTANCE|α10|
|リリア|発動POIS_BALL_AND_RESITANCE|α9|
|グラムドリング|レア度20→12、+1知賢探道、耐:邪ト巨、*オ*、易|α10|
|オルクリスト|レア度20→12、+3器隠探赤、耐:邪ト巨、*オ*|α9|
|ノートゥング|レア度90→12、+4腕探、麻|α9|
|フォラスギル|階15→10、(2d6)、追加上位耐性|α9|
|カレス・アストリアグ|階15→10、*動人オト巨竜*|α9|
|暁の剣|レア度120→40|α9|
|アンドゥリル|レア度40→20|α10|
|アランルース|レア度45→20、+4賢器、対:悪オト巨、属性:冷、視祝遅、ESP:悪オト巨、LITE_2|α10|
|クリスドゥリアン|レア度15→10、クレイモアに変更(3d8)、10kg|α10|
|ゴンドリカム|探索、水耐性、追加上位耐性|α11|
|手品師|修復時にボーナス？||
|オソンディア|階20→12、(4d5)、+3腕魅、対:死、*巨*、追加上位耐性、LITE_3|α9|
|ティル=イ=アルク|階20→12、レア度15→8、(3d5)、+2腕知耐、維:腕知耐、急回復、LITE_2|α9|
|グングニル|レア度50→30|α9|
|ロンギヌス|レア度40→25、属性:炎電、易、LITE_3|α9|
|ロサラング|階30→15、レア度15→8、(3d8)、+2腕器掘、耐:動、*オト*、耐:恐、麻、発動HERO|α9|
|バルクケレド|麻視、LITE|α9|
|ウルモ|レア度90→35、水耐性|α11|
|アヴァビア|階45→15、属性:焼凍電、耐:炎冷電閃、追加上位耐性|α10|
|トゥアミル|(+10,+16)、耐:冷閃暗、易|α9|
|蜻蛉切|レア度70→50|α9|
|ガエボルグ|レア度90→30|α9|
|二郎真君|レア度45→25|α9|
|アイグロス|レア度45→30|α9|
|天の沼矛|(3d8)、水耐性、宝物庫卒業|α1|
|トティラ|+1攻隠掘、LITE_2|α9|
|稲妻の拳|レア度38→20、発動BA_ELEC_2|α9|
|血流の刺|レア度30→15、麻視急|α9|
|焔の星|(3d6)、+2腕、耐:火恐、LITE_3|α9|
|サムソン|レア度20→8|α9|
|隠者の紫|階40→20、レア度40→30|α10|

**Reply by deskull-m** [reply]
レア度は低下はもっと見つかっていいと思うものばかりですし、強化もぱっと見てやり過ぎな様子はないと思います。Discordでdis-氏も言っていた通り、上等なエゴ武器がこれを吹き飛ばす勢いになる例もあるくらいですから、この位は問題内でしょう。

**Reply by gold-onion** [reply]
調整をお願いしたいアーティファクトがいくつかあります。

アヴァビア　α９調整でも威力に修正が入ってないので40階なのに宝物庫クラスより弱い
　ベースアイテムが45階で生成レベルを下げるのは効果が薄いので威力を上げる必要がある

アランルース　現状でグラムドリング等より若干弱いと思われるがα９調整から漏れたためさらに差が開いた

隠者の紫　ダイス目が低すぎでスレイの効果が弱い
　ここから威力を上げないなら同じテレパシー武器のセオデン王ぐらいの階層が適当かも

手品師　希少度を3～5ぐらいにしてほしい　修復時に能力2個継承してほしい

ロングソードのデーモンベーン　希少度を3～5ぐらいにしてほしい

**Reply by iks3** [reply]
アヴァビアの階は調整し忘れですので、次回実装したいと思います。
その他についても考慮します。(アランルースは存在を忘れていました)

**Comment by iks3** [comment]
### 防具編
|名称|変更点|対応|
|---|---|---|
|ヒスロミア|発動PHASE_DOOR|α10|
|サルケットス|(+~~7~~3,+3)、+3器隠、耐:酸~~冷~~恐破劣|α11|
|ヒムリング|耐:毒獄沌劣|α10|
|インカヌス|追加上位耐性|α10|
|カスパニオン|耐:酸毒乱轟、麻|α10|
|バテレン鎧|[23,22]、耐:炎轟沌|α10|
|ジャイルズ|レア度20→5|α10|
|刀返し|耐:水時、麻||
|ホルコルレス|+3知賢隠探、維:知賢、視、発動ID_PLAIN|α10|
|シンゴル|レア度90→30|α10|
|ソロンギル|レア度20→8、+2器魅赤隠探|α10|
|コランノン|レア度10→8、+5隠、耐:酸冷、麻|α10|
|ルシエン|レア度40→15|α10|
|トゥオル|レア度40→10、水耐性|α11|
|ギル=ガラド|レア度4→2、耐:酸炎冷電閃、LITE_2|α10|
|エアレンディル|レア度6→3、LITE_3、水耐性|α11|
|ペルセウス|+2腕、耐:酸冷毒恐、維:腕、麻反、LITE_2、追加上位耐性|α10|
|インドラ|レア度50→30、耐:冷閃盲轟、LITE、発動BA_ELEC_3|α10|
|センゲル|追加上位耐性|α10|
|槌手王|耐:酸恐因|α10|
|ベルシエル|-5腕器耐、ESP:感邪無善、発動DETECT_ALL_MONS|α10|
|ホルヘンネス|+2知賢赤探、追加上位耐性|α10|
|ヌメノール|耐:閃暗轟破、追加上位耐性、発動PROT_EVIL|α10|
|カンミスリム|易、LITE_3|α10|
|パウアハッハ|レア度20→10、+2腕力、耐:火轟|α10|
|パウアネン|レア度20→10、+2賢さ、耐:酸劣|α10|
|パウアニンメン|レア度20→10、+2知能、耐:冷破|α10|
|パウラエゲン|レア度20→10、+2器用、耐:電暗|α10|
|ガラス|レア度40→20|α10|
|フローラ|レア度25→15|α10|
|鉄ゲタ|レア度15→10|α10|

**Reply by HiddenNameZ** [reply]
サルケットス・ヒムリングは現状でも悪くない性能なので強化なくてもいいと思います
ベルシエルはxangを見習って実用性を与えるならPVAL-3でもいいんじゃないでしょうか 兜枠は競合相手も多いので
ペルセウス盾はwikiのコメント欄が元ネタですが、メドゥーサの首が付けてあるので殺戮修正方面でも良いと思います 盾★で殺戮修正付与するのがないですし(PYEC除く)

**Reply by oguchi2918** [reply]
装備制限職やデュアル用として採用率が高い★サルケットスは強化し過ぎかと。
隠密は★ヒスロミアと被りますし、殺戮修正or耐性追加のどちらか一方だけでも十分かと。
★インドラですが、メイジ系の古い城報酬なのでレア度は高い方(60or70?)がよいかと思います。
+5魅の代わりに+5道にしてメイジ系の特権だけど実は戦士系で引いた方が嬉しいという立ち位置はどうでしょ。

**Comment by iks3** [comment]
### 弓編
|名称|変更点|対応|
|---|---|---|
|バルド|+3腕器、対竜、追加上位耐性|α10|

### その他編
|名称|変更点|対応|
|---|---|---|
|帯魔力|+2知魅探赤道、易|α10|
|ゴゴ|維:知賢器|α10|
|フラキア|麻|α10|

**Reply by gold-onion** [reply]
ゴゴは古い城報酬にもかかわらず肝心の耐久修正が無いためものまね師があまり使ってないという話が
PVALが4なので耐久を単純に足すと強すぎるかも

**Reply by chikatei** [reply]
ゴゴのペンダントは装備時にものまねや倍返しの失敗率が下がるという特性があったらものまね師にとって嬉しい装備になると思いました。特に倍返しは失敗率が最低でも15%も残るので。

**Reply by Hourier** [reply]
一般的なものまねは＠のパラメータ (レベル他)と失率に非線形な関係があったので、かなり雑な調整ですが失率が下がるようにしました
クラスレイシャルである倍返しの最終失率は9% (取り敢えず1桁)になるように調整しました
α25のリリースをお待ち下さい
強すぎる/弱すぎる調整でしたらα26以降に再調整します

**Comment by iks3** [comment]
[252] ★グレート・ソード『サンブレード』 (3d6) (+16,+20) [+10] / The Two-Handed Sword 'Sun Blade' について
I:23:25:3
W:30:150:180:65000
P:0:3d6:16:20:10
F:INT | FORCE_WEAPON
F:SHOW_MODS
F:SLAY_EVIL | SLAY_UNDEAD
F:FREE_ACT | LITE

**Reply by iks3** [reply]
ベース武器をバスタード・ソード、重さを4.0kgにして調整 (D&Dのやつがそのような設定らしい)
(5d4) (+16, +16) (+3) {+知賢探赤;麻視明祝[炎|焼/X邪死} 発動:眩しい光

I:23:21:3
W:40:150:80:80000
P:0:5d4:16:16:0
F:INT | WIS | INFRA | SEARCH | HIDE_TYPE | SHOW_MODS
F:BRAND_FIRE | KILL_EVIL | KILL_UNDEAD | SH_FIRE
F:SEE_INVIS | FREE_ACT | BLESSED | LITE_3 | ACTIVATE
U:BLINDING_LIGHT

**Reply by iks3** [reply]
理力については、別途元ネタっぽい武器を提案していただく方がよい、という方向になりました。
(光魔の杖、ライトブリンガー、ルーンアクス等)

**Reply by deskull-m** [reply]
X邪は慎重にすべきかもしれんですが、5d4ですし結局殴らんといかん職である以上こんくらいはあってもいい気はしますな。

**Reply by meetname** [reply]
横から失礼しますが、★二天一流開祖宮本武蔵のカタナと★二天一流開祖宮本武蔵の脇差しはそのままなのでしょうか？

**Reply by iks3** [reply]
> ★二天一流開祖宮本武蔵のカタナと★二天一流開祖宮本武蔵の脇差し
とりあえずα9の時点ではそのままで、(ほかの★も含めて)これから考えるかな、といった感じです。

**Reply by iks3** [reply]
サンブレード、α9で実装しました。

**Comment by iks3** [comment]
[171] ★二天一流開祖宮本武蔵のカタナ (3d4) (+28,+17) / The Katana of Musashi
[172] ★二天一流開祖宮本武蔵の脇差し (2d4) (+28,+17) / The Wakizashi of Musashi
以上2つについて

**Reply by HiddenNameZ** [reply]
とりあえずX人をつけるのはどうでしょう
あと剣術家なら追加攻撃+1がつくとか

**Reply by iks3** [reply]
α11で調整しました。

**Comment by HiddenNameZ** [comment]
自分としては★カットラス『ゴンドリカム』と★首切りソード『クリスドゥリアン』について強化が欲しいです
前者は現状ほぼ防衛者エゴの下位互換となってしまっているので、毒耐性+追加耐性くらいはあってもいいかと
後者は現状弱いわけではないですが、パッとしなさが凄いので…

**Reply by iks3** [reply]
α10～11で調整しました。
クリスドゥリアンはクレイモア化で微強化＋軽量化。
ゴンドリカムは水耐性と上位追加耐性つけて様子見です。

**Comment by chiku-tamariba** [comment]
[171]★ロング・ソード『エクスカリバー』 (5d5) (+21,+19) [+5]について強化調整を提案します。

以前より希少度の割にパっとしない性能で5d5の当たりの外れの方等と言われる事もあり、★サンブレードや★ドンキホーテのヘヴィ・ランス等この武器の対邪ダメージを越える古い城報酬が追加され、更に微妙な立ち位置になってしまったと思います。
具体的な提案としましては切れ味を無くす代わりにX邪デ死として対邪ダメージ71.5点、対悪魔アンデッドには94点の武器として調整するのは如何でしょうか？
対邪70点台は現状デメリット付きの二つしか存在しない為、古い城報酬とヴォーパル、如意棒等トップ層との中間の立ち位置として丁度良いかと思います。
強化版★サンブレードのような性能になりますが、元ネタのひとつであるファイナルファンタジーでの両者の性能的にも問題無いかと思われます。

どうぞご検討お願い致します。

**Reply by chiku-tamariba** [reply]
ID[176]の間違いです。失礼致しました。

**Reply by deskull-m** [reply]
たしかにちょっとネームバリューに劣る感じはしますね、そんな感じの調整で行って良い気はします。

**Reply by chiku-tamariba** [reply]
ありがとうございます！また、この場をお借りして…最近の更新ラッシュ本当にワクワクしております。開発に携わっている皆様に感謝です！

**Reply by Aquashrimp** [reply]
discordで、#198 の魔法防御上昇フラグが実装されたら付けても良いんじゃないかという話が出てました

**Comment by chiku-tamariba** [comment]
反テレポート付き★武器の緩和調整提案
強力でありながらその大きすぎるデメリットにより狂戦士以外では使われていないであろういくつかの★武器についてデメリットの緩和を提案します。

[158]★アヌビス神のカタナ (3d4) (+28,+22) [-30]について
★レイピア『シルバーチャリオッツ』との二刀流で反テレポートの消去(スポイラーWikiのコメント欄のアイディアより)
強力ですが、最終戦では二刀流ペナルティの緩和は無い、相方のシルバーチャリオッツが最終武器としてはやや弱い、オーラダメージが痛い、等の理由からバランスブレイカーとまではいかないかと思います。

[144]★カタナ『村正』 (10d5) (+0,+0) [-20]について
発動：妖刀をなだめる(確率50%で反テレポート, 経験値吸収, 反感, 太古の怨念,反魔法のいずれかを消去、残りの50%で武器が壊れる)に変更
面倒な処理かと思いますので、現在の処理の延長で職業剣術家なら反テレポート、反魔法の消去としても良いと思いますが、どうせなら色々な職で使える可能性があった方が楽しいかと思います。ただ、現在の発動は武器の性能も含め完璧な原作準拠になっているのでWizファンの方には難色を示されるかも知れません。
恐らくヴォーパルを超える火力になってしまうと思いますが、希少度200をツモった上で1/2のギャンブルに1～5回勝つという幸運に恵まる必要があるのでご褒美として許容されるかなと

従来の狂戦士用の武器としての立ち位置は変えずに他職で使いやすくなるよう考えてみました。
レアな武器をせっかくツモっても即博物館行きというのは悲しいので、ご検討頂ければ幸いです。

**Reply by deskull-m** [reply]
正直、この辺のペナルティが重い武器は止むを得ないと思われますし、実装が面倒な案も多いのでこの辺りは考えさせて下さい。

**Comment by HiddenNameZ** [comment]
### 超能力者の古い城報酬について
現在、超能力者の古い城報酬は★インドラの硬革帽子となっていますが、
- 超能力者は古い城クリア前から、電撃の二重耐性(LV30)と賢さ維持(LV20)が得られる
- 超能力者は盲目状態でもテレポートを含めた超能力を使えるうえ、古い城クリア前からテレパシーの一時耐性(LV25)、そして古い城クリア前後でテレパシー(LV40)が得られるので、盲目耐性があまり嬉しくない

など、あまりふさわしい報酬とは思えません。似たような魔法システムの鏡使いの報酬が★ヌメノールの宝冠なのでそちらに変更してもよいのではないでしょうか。器用さ上昇もデュアルである超能力者にありがたいように思われます…と思ったのですが、★ヌメノールの宝冠って賢さ上昇がないんですね…なのでPVALに賢さを与えてもよいかと思います(元々は個別で立てようと思ったのですが、修正案も含めるのでこちらに)

**Reply by gold-onion** [reply]
報酬は★プラグがいいかなと思ったが、これだと強すぎるかな

**Reply by HiddenNameZ** [reply]
アリではあると思いますがやはり永遠の呪いというのは…むしろ呪術師向けじゃないかなぁ…

**Reply by chikatei** [reply]
超能力者は腕耐賢速確保するのが大変なので、インドラよりはプラグのほうが使う機会はありそう。永遠の呪い付きだから拾って即装備するかは判断の余地がありますし、強すぎるということは無いと思います。

**Comment by Aquashrimp** [comment]
★スランドゥイルを拾って*鑑定*したら、追加耐性もr盲が選ばれたためか耐性がr盲のみ、ということがありました
種族などの耐性と被るのは致し方ないですが、追加耐性を装備自体の耐性と被らないようにすることは可能でしょうか？

**Comment by daradarach** [comment]
新装備フラグ - 魔法防御上昇 #198 が実装の折には元ネタ考慮で★運命のオーブにも付与を希望します。
その他の★にも適宜与えることを考えてもいいのかもしれないです。

**Comment by HiddenNameZ** [comment]
### 一部のAFのベースアイテムの和名/英名の不一致について、
現在、★射手バルド王の黒い矢のベースアイテム名が和名では「矢」、英名では"black arrow"となっています。
自動拾いの性質を考えると、和名の方を「黒い矢」、AFとしての名前を「★射手バルド王の」とした方が良いかと思います。

また、和名ベースアイテムは「石」だが、★パランティアの石の英名ベースアイテムは"crystal ball"、★浮遊石の英名ベースアイテムは"levitation stone"となっています。
自動拾いの都合から、ベースアイテムについては前者は「水晶玉」、後者は「飛行石」としても良いのではないでしょうか。
(AF名は「パランティアの石」(FULL_NAMEフラグ)と「ラピュタの」で)

**Comment by RRyukkuriRyu** [comment]
★ヱヅソンの白熱灯強化案
現状★光源どころか不朽の眼、灼熱エゴにも劣りかねない性能なので強化案を出します。

N:7:ヱヅソンの
E:of Yeduson
I:39:3:3
W:40:20:5:20000
P:0:1d1:0:0:0
F:INSTA_ART | INFRA | RES_FIRE | RES_ELEC | SH_FIRE | SH_ELEC | LITE_3 |
D:$The Incandescent Light of Yeduson 
D:$Thomas Alva Yeduson the "Wizard of Menlo Park" 
D:$patented 1,093 inventions.  
D:$Inventor of the incandescent lightbulb.
D:「非常に進んだ科学は魔術と見分けがつかない。」

灼熱エゴにr火が付いたのに習ってr火電を付けました。元素耐性が一部抜けてしまう状況はそれなりにあるので、そういう穴を光源枠で埋められるようになるのは大きいと思います。

**Reply by gold-onion** [reply]
どちらかと言うと保温用電球のイメージで耐火耐電より耐冷なのでは
他には知力補正とか青魔道師がラーニングしやすくなるとか

**Comment by daradarach** [comment]
PYECは元だと魔法防御能力があるのでr呪あっていい気がします。
Nethack由来のアーティファクトに対して大安売りな感じもしちゃいますが。

**Comment by Hourier** [comment]
OSDN 時代の★類を軒並み実装し忘れていました
それらが片付くまで一旦新規の受付を停止します
再開時は改めて告知しますので、各自のPC内アイディア温めておいて頂けますと幸いです
よろしくお願いします

**Comment by HiddenNameZ** [comment]
ここに投稿するべきかやや疑問ですが、スーパーモンキー『孫悟空』のドロップ品として★鉄棒『如意金葹棒重一万三千五百斤』はどうでしょうか？ 1％くらいで
ブランドボウが許されるなら…中盤までは装備しづらいですし

**Comment by gitgita6** [comment]
マジックベーンの希少度を上げて欲しいです

発動も含めると殆どの職で重宝する有能さがLV30希少度40で結構入手できる印象なので、同じLV30スレイヤーの希少度 60以上で良いと思います（このさじ加減は要議論でしょうが）

魔法属性自体を弱化させる方向よりは希少度調整のみの方が面白いと思います

**Comment by meetname** [comment]
斧が武器としてイマイチすぎます。
会心の一撃時の重量ボーナスが大きいとはいえ、重く、攻撃回数が増えにくいため採用しにくいです。
固定アーティファクトは★ドワーフのロッホアーバー・アックスと★フーリン王あたりは強いですが、アクセサリ向きのものが多い。そしてアクセサリとしても重い。
重量に能力が追い付いてくる頃には大抵斧以外の固定アーティファクトが充実しているので、序盤～中盤向けの性能が多い斧の固定アーティファクトは候補から外れてしまいがちです。
固定アーティファクトですらこんな状態なので、強いエゴが付いても同じような感じです。
★鉄棒『如意金葹棒重一万三千五百斤』は比較対象としては強すぎますが、★巨人夸父といったかなり重い武器にも
重量威力ともに追い抜かれてしまっています。
斧全般のダイス増加などどうでしょうか。
